OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -21,9 +21,34 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	[Flags]
+	public enum FlightDynamic
+	{
+		None = 0,
+		MoveIntoShroud = 1,
+		Slide = 2,
+		Hover = 4,
```

Comment:
I thought the plan was to replace this with an `IdleSpeed` setting?

Reply:
Maybe, but @pchote basically said we shouldn't open the can of worms called idle speed before next release, so if we want FlightDynamics before that, this is needed at least as stop-gap solution.